clusters/app.ci/supplemental-ci-images/assisted-ci-tools/postgresql-is.yaml

```diff
@@ -0,0 +1,11 @@
+kind: ImageStream
```

Comment:
I think it should be updated in this file: https://github.com/openshift/release/blob/master/core-services/image-mirroring/_config.yaml#L448-L449

https://docs.ci.openshift.org/docs/how-tos/external-images/

Reply:
Instead ? this is the way we used go-swagger, swagger_codegen until now

Reply:
then it would be good to consolidate in one way, `clusters/app.ci/supplemental-ci-images/assisted-ci-tools` looks fine because we don't need DPTP to approve